src/components/StatsUsageChart.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <base-line-chart :data-sets="dataSets" :labels="labels" :key="season.id" />
```

Comment:
I threw a `key` on this because data wasn't updating reactively when setting another season but I think that's just because I was setting the chart data in component `data()` nonreactively instead of as a comptuted prop
```suggestion
  <base-line-chart :data-sets="dataSets" :labels="labels" />
```